### PR TITLE
feat: Add zio-schema-quickstart

### PR DESCRIPTION
**Pull request by Yummy-Yums**

### CODE REVIEW

**Comment by Yummy-Yums** [comment]
test passes 
#21 

image

File: zio-quickstart-schema/src/main/scala/dev/zio/quickstart/Person.scala [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+object Person {
+    implicit val schema : Schema[Person] = DeriveSchema.gen
+}  
+
+
+// one can choose the detailed way of validation or use annotations
+
+//case class Person(name: String, age: Int)
+
+// object Person {
```
Comment (khajavi) [inline zio-quickstart-schema/src/main/scala/dev/zio/quickstart/Person.scala:25]: Commented code is hard to maintain! I believe you can find a way to show both solutions.

**Comment by Yummy-Yums** [comment]
Added new model PersonWithEmail for manual schema validation 

image

**Review by khajavi — APPROVED** [review]

**Comment by khajavi** [comment]
@Yummy-Yums Thanks for your contribution!

**Pull request merged**